crates/orderbook/openapi.yml

```diff
@@ -772,12 +802,11 @@ components:
             For ethflow orders - order that are placed onchain with native eth -, this field
             contains a struct with two variables user_valid_to and is_refunded
           $ref: "#/components/schemas/EthflowData"
-        onchainUser:
+        onchainOrderData:
```

Comment:
Same nitpick regarding the property case

Reply:
Actually, realized is the other way around, this field is fine.
The snake_case on the new props is what's breaking the consistency